x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -1330,4 +1330,69 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'mac.advanced.kernel.fileaccess',
+    first_supported_version: '8.11',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.mac.advanced.kernel.fileaccess',
+      {
+        defaultMessage: 'Sends file access events to the stack. Default: false',
+      }
+    ),
+  },
+  {
+    key: 'mac.advanced.events.image_load',
+    first_supported_version: '8.11',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.mac.advanced.events.image_load',
+      {
+        defaultMessage:
+          'Collects image load events and use them for behaviour protection. By default, does not send events to the stack. Default: true',
+      }
```

Comment:
From our meeting, we agreed image load events collection (but not writing to Elastcsearch) will be enabled if Malicious Behavior is enabled.

This option is based on `mac.advanced.events.process` and will force image load events off even if something else in policy (like Malicious Behavior or the option below) would cause them to be enabled.

```suggestion
      'xpack.securitySolution.endpoint.policy.advanced.mac.advanced.events.image_load',
      {
        defaultMessage:
          'A value of 'false' overrides other config settings that would enable kernel image load events. Default: true.',
      }
```